ui/locales/source.json

```diff
@@ -134,6 +134,10 @@
     "description": "Label for a button used to skip ahead to the live broadcast.",
     "message": "Skip ahead to live"
   },
+  "SUBTITLE_FORCED": {
+    "description": "Label used to identify a forced subtitle.",
```

Comment:
I would suggest a bit more description.  This is used by the translation team to determine how to translate the text.  They may not know about forced subtitles.  How about: `Label used to identify a subtitle track that is forced to be shown`?

Reply:
Sounds better to me, I'm going to change it.